ktor-shared/ktor-serialization/ktor-serialization-jackson/jvm/test/ClientJacksonTest.kt

```diff
@@ -3,6 +3,10 @@
  */
 
 import com.fasterxml.jackson.annotation.*
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.smile.SmileFactory
+import com.fasterxml.jackson.module.kotlin.jacksonTypeRef
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
```

Comment:
One more minor comment :)
Could you please replace imports with wildcard according to our code style?